ci/travis/after_success-release-linux.sh

```diff
@@ -32,11 +34,12 @@ sudo pip install twine || exit
 echo "Creating distribution files..."
 # This release build creates the source distribution. All other release builds
 # should not.
-python setup.py sdist bdist || exit
+python setup.py sdist bdist bdist_wheel || exit
```

Comment:
Following the recommendation of the official [Python Packaging Guide](https://packaging.python.org/en/latest/distributing.html#uploading-your-project-to-pypi).

> (Recommended): Use twine
>
>     twine upload dist/*
> The biggest reason to use twine is that python setup.py upload (option #2 below) uploads files over plaintext. This means anytime you use it you expose your username and password to a MITM attack. Twine uses only verified TLS to upload to PyPI protecting your credentials from theft.